libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.html

```diff
@@ -56,11 +58,19 @@
 
       <sky-datepicker>
         <input
+          class="sky-form-control"
```

Comment:
This class causes input box easy mode to not kick in. Without it - the id that we were setting wasn't being applied.

I think we should refactor this eventually to use easy mode - but didn't want to add that on to this at this point. 

The class does not cause any styling changes.